visualizations/frontend/other/TPdfTable.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="relative overflow-auto">
+    <t-loading-spinner v-if="loading" position="relative" />
+    <slot name="header" :loading="loading"></slot>
+    <div :class="containerClass">
+      <table class="w-full table-fixed">
+        <thead>
+          <th v-for="column in columns" :key="column" :class="columnsClass">
+            <slot :name="'col-' + column" :column="column">
+              {{ column }}
+            </slot>
+          </th>
+        </thead>
+        <tbody>
+          <tr v-for="(row, index) in rows" :key="'row-' + index">
+            <td
+              v-for="column in columns"
+              :key="column"
+              class="break-all"
+              :class="rowsClass"
+            >
+              <slot :name="'row-' + column" :row="row">
+                {{ row[column] ? row[column].rendered : "" }}
+              </slot>
+            </td>
+          </tr>
+        </tbody>
+        <div v-if="!loading && !rows.length" class="text-center">No Data</div>
+      </table>
+    </div>
+    <slot
+      name="footer"
+      :rows-limit="rowsLimit"
+      :rows-total="rowsTotal"
+      :loading="loading"
+    >
+      <div class="pt-4 text-right">
+        Showing {{ rowsLimit }} of {{ rowsTotal }}
+      </div>
+    </slot>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    hiddenColumns: {
+      type: Array,
+      default: () => [],
+    },
+    dynamicColumns: {
+      type: Array,
+      default: () => [],
+    },
```

Comment:
Do we even need a concept of dynamicColumns in the PDF, since you cannot actually modify columns in a PDF? Can we simply use the URL params and pass them as the columns to display in the PDF?

Reply:
That was my original intention but original table uses this weird syntax that I am not really sure about.

Basically the value in URL is sometimes different from value in html.
<img width="303" alt="Screenshot 2023-06-13 at 5 36 55 PM" src="https://github.com/topcoat-data/topcoat-public/assets/43262405/57fb9fa2-55a2-4a50-a920-aa5ba65233bc">